src/text/raw_text_writer.rs

```diff
@@ -400,7 +395,8 @@ impl<W: Write> RawTextWriter<W> {
     pub fn add_annotation<A: AsRawSymbolTokenRef>(&mut self, annotation: A) {
         // TODO: This function currently allocates a new string for each annotation.
         //       It will be common for this text to come from the symbol table; we should
-        //       make it possible to pass an Rc<str> or similar when applicable.
+        //       make it possible to pass an Arc<str> or similar when applicable.
+        //       See: https://github.com/amazon-ion/ion-rust/issues/496
```

Comment:
🗺️ I noticed this `TODO` and opened an issue (#496) to track eventually implementing this optimization.